games-simulation/openrct2/openrct2-0.0.7.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils gnome2-utils
+
+DESCRIPTION="An open source re-implementation of RollerCoaster Tycoon 2"
+HOMEPAGE="https://openrct2.website/"
+MY_REPO="https://github.com/OpenRCT2/OpenRCT2"
+if [ "${PV}" == "9999" ]; then
+	EGIT_REPO_URI="${MY_REPO}.git"
+	EGIT_BRANCH="develop"
+	inherit git-r3
+	SRC_URI=""
+else
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="${MY_REPO}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/OpenRCT2-${PV}"
+fi
+SRC_URI+="
+https://github.com/OpenRCT2/title-sequences/releases/download/v0.0.5/title-sequence-v0.0.5.zip -> ${PN}-title-sequence-v0.0.5.zip
+"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+twitch +multiplayer opengl truetype libressl test"
+
+# This is needed because of this bug: https://github.com/OpenRCT2/OpenRCT2/issues/5469
+REQUIRED_USE="multiplayer? ( twitch )"
+
+RDEPEND="
+	twitch? ( net-misc/curl[ssl] )
+	multiplayer? (
+		libressl? ( dev-libs/libressl:0= )
+		!libressl? ( dev-libs/openssl:0= )
+	)
+	opengl? ( virtual/opengl )
+	truetype? (
+		media-libs/sdl2-ttf
+		media-libs/fontconfig
+	)
+	media-libs/libsdl2
+	|| (
+		(
+			>=media-libs/speex-1.2.0
+			media-libs/speexdsp
+		)
+		(
+			<media-libs/speex-1.2.0
+			>=media-libs/speex-1.2_rc1
+		)
+	)
+	>=dev-libs/jansson-2.5
+	>=dev-libs/libzip-1.0
+	media-libs/libpng:=
```

Comment:
libpng slots 1.2, 1.5 do not supply the headers. If you're building from source, then you need to require slot 0.